src/main/java/org/mskcc/cbio/portal/dao/DaoClinicalData.java

```diff
@@ -50,9 +50,12 @@ public final class DaoClinicalData {
     public static final String SAMPLE_TABLE = "clinical_sample";
     public static final String PATIENT_TABLE = "clinical_patient";
 
-    private static final String SAMPLE_INSERT = "INSERT INTO " + SAMPLE_TABLE + "(`INTERAL_ID`,`ATTR_ID`,`ATTR_VALUE` VALUES(?,?,?)";
-    private static final String PATIENT_INSERT = "INSERT INTO " + PATIENT_TABLE + "(`INTERNAL_ID`,`ATTR_ID`,`ATTR_VALUE` VALUES(?,?,?)";
+    private static final String SAMPLE_INSERT = "INSERT INTO " + SAMPLE_TABLE + "(`INTERNAL_ID`,`ATTR_ID`,`ATTR_VALUE`) VALUES(?,?,?)";
+    private static final String PATIENT_INSERT = "INSERT INTO " + PATIENT_TABLE + "(`INTERNAL_ID`,`ATTR_ID`,`ATTR_VALUE`) VALUES(?,?,?)";
```

Comment:
not an issue introduced here, but we might want to improve it: the name `SAMPLE_TABLE` is confusing since we do have a table called `sample` in the DB. It would be clearer to name this `SAMPLE_ATTRIBUTES_TABLE` for example. The naming issue led me to misinterpret the rest of the code added here...

Reply:
indeed